src/features/session_trace/aggregate/index.js

```diff
@@ -99,7 +100,7 @@ export class Aggregate extends AggregateBase {
 
         if (prevMode === MODE.ERROR && this.#scheduler) {
           this.trimSTNs(ERROR_MODE_SECONDS_WINDOW) // up until now, Trace would've been just buffering nodes up to max, which needs to be trimmed to last X seconds
-          this.#scheduler.runHarvest({ needResponse: true })
+          this.#scheduler.runHarvest({})
```

Comment:
Why having to pass empty object in? `runHarvest` needs fixing its opts param if undefined

Reply:
functionally identical, doesn't matter to me.